src/docs/background/23-how-capabilities-work/index.md

```diff
@@ -0,0 +1,117 @@
+be a complex and daunting endeavour. Fulfilling these requirements
+means that you need to be collecting a variety of information from
+your providers in order to be able to process payments. Stripe has a
+mechanism for controlling the amount of information collected from providers called
+[Capabilities](https://stripe.com/docs/connect/capabilities-overview).
+
+Capabilities are a collection of settings that can be requested for
+each provider. What Capabilities are required determines what
+information Stripe requires to be collected from the providers.
+
+## Changes to Capabilities framework
+
+Stripe is introducing mandatory changes to the Capabilities framework.
+In the beginning of 2020, Stripe will enforce that Capabilities are
+required for all accounts, existing and new. Stripe’s deadlines for
+reacting to these changes are:
+- 20th of February, 2020 for customers in the US.
+- 31st of March, 2020 for the rest of the world.
+- In practice, there might be small variations for these depending on the
+ country and activity of the marketplace.
+
+After requesting new Capabilities, your providers need to most likely
+provide additional verification information to Stripe and you need to
+update your provider onboarding and payment information updating
+processes in your marketplace application.
+
+## How should you handle the changes to Capabilities framework?
+
+For Flex payments to operate smoothly, your accounts need to have the
+`card_payments` and `transfers` Capabilities required and activated.
+The `card_payments` will enable the providers to accept payments by
+card and the `transfers` will allow the platform account to move money
+to connected accounts. Requesting `card_payments` will enforce also
+requesting the `transfers` capability.
+
+Requesting Capabilities can be done during [Stripe
+Account](https://www.sharetribe.com/api-reference/marketplace.html#stripe-account)
+[create](https://www.sharetribe.com/api-reference/marketplace.html#create-stripe-account)
+and
+[update](https://www.sharetribe.com/api-reference/marketplace.html#update-stripe-account)
+calls. Activating the Capabilities happens by providing identity
+verification information to Stripe either by using [Stripe Connect
+Onboarding](https://stripe.com/en-fi/connect/onboarding), or [account
+tokens](https://stripe.com/docs/connect/account-tokens)
+during the create and update calls. What information is required for
+Capability activation is dependent on provider account and activity.
+The specific information can be determined from `stripeAccountData` in
+[Stripe
+Account](https://www.sharetribe.com/api-reference/marketplace.html#stripe-account)
+resource. The `stripeAccountData` attribute contains the related [Stripe Account
+Object](https://stripe.com/docs/api/accounts/object) where the fields
+`payouts_enabled` and `requirements` can provide you with valuable information.
+
+How to implement these changes depends on when you have subscribed to Flex.
```

Comment:
How about:

Depending on when you have subscribed to Flex, you may need to take action and implement changes to your marketplace's provider onboarding flow.